### PR TITLE
[WIP]: MH-11833: Fix mimetype for legacy MH 1.3 Capture Agent

### PR DESCRIPTION
**Pull request by ziegenberg**
Supposedly fixes https://opencast.jira.com/browse/MH-11833

### CODE REVIEW

File: modules/common/src/main/resources/org/opencastproject/util/MimeTypes.xml [hunk 1]
```diff
@@ -19,6 +19,11 @@
     <Description>JavaScript Object Notation</Description>
     <Extensions>json</Extensions>
   </MimeType>
+  <MimeType>
+    <Type>application/octet</Type>
+    <Description>MH 1.3 CaptureAgent MPEG media</Description>
+    <Extensions>mpeg,mpg,m1s,m75,m15,mpm,mpv</Extensions>
```
Comment (lkiesow) [inline modules/common/src/main/resources/org/opencastproject/util/MimeTypes.xml:25]: Shouldn't `*.mpg` or `*.mpeg` have a mime type like `video/mpeg` or `video/mpg`? Not sure about the rest…

Comment (gregorydlogan) [reply modules/common/src/main/resources/org/opencastproject/util/MimeTypes.xml:25]: Agreed.  I don't think I've even see the other formats, aside from the occasional mpv.  @ziegenberg, is this something you're running into regularly?

Comment (lkiesow) [reply modules/common/src/main/resources/org/opencastproject/util/MimeTypes.xml:25]: `mpv` is my default media player ;-D

Comment (ziegenberg) [reply modules/common/src/main/resources/org/opencastproject/util/MimeTypes.xml:25]: I'll recheck. This fix was some suggestion from a third party.

Comment (lkiesow) [reply modules/common/src/main/resources/org/opencastproject/util/MimeTypes.xml:25]: Reading through the Jira ticket, using `<Type>video/mpeg</Type>` should work

**Comment by lkiesow** [comment]
Any progress on this? If there is no immediate plan to continue working on this, I suggest that we decline this pull request. Keeping this open forever will not help anyone.

**Comment by ziegenberg** [comment]
Yes, I plan to proceed with the issue. It just the case that I have no time the next two weeks, to get a test installation up and running.  But afterwards I will continue.

**Comment by lkiesow** [comment]
Did you accidentally merge develop into this pull request?

**Comment by ziegenberg** [comment]
No, that merge was intentional. I just changed the base branch.

**Comment by lkiesow** [comment]
Looking at this again, I noticed that the file extensions have already been included by now:
[MimeTypes.xml#L390](https://github.com/opencast/opencast/blob/develop/modules/common/src/main/resources/org/opencastproject/util/MimeTypes.xml#L390)

**Comment by staubesv** [comment]
Hi @lkiesow The problem with this seems to still persist. Could you please have a look at the comments on https://opencast.jira.com/browse/MH-11833 and reply to Andy?

**Pull request closed without merging**